.github/workflows/onPR.yml

```diff
@@ -20,3 +20,9 @@ jobs:
       - run: npm ci
       - run: npm run build 
       - run: npm run test
+      - run: npm run make-badges:ci
+      - run: npm run coverage:codecov
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v2
+        with:
+          flags: coverage
```

Comment:
Please add a new line character at the end of the file

Reply:
fixed in https://github.com/the-ai-team/razor/pull/19

Reply:
merged #19 into this